packaging/package_all.sh

```diff
@@ -33,56 +38,82 @@ function _move_debs() {
   if ls ../*.deb 1> /dev/null 2>&1; then
     mv ../*.deb ${SCRIPT_PATH}/${DEBS_OUTPUT_DIR}/
   fi
-  if ls ../*.ddeb 1> /dev/null 2>&1; then
-    mv ../*.ddeb ${SCRIPT_PATH}/${DEBS_OUTPUT_DIR}/
-  fi
 }
 
 function _execute_build() {
+#  bloom-generate rosdebian --os-name ubuntu --os-version focal --ros-distro foxy --place-template-files
   bloom-generate rosdebian --os-name ubuntu --os-version focal --ros-distro foxy -i "${GIT_VER}"
   sed -i 's/^\tdh_shlibdeps.*/& --dpkg-shlibdeps-params=--ignore-missing-info/g' debian/rules
   fakeroot debian/rules clean
   fakeroot debian/rules "binary --parallel"
 }
 
+function _make_deb() {
+  local pkg=$1
+  cat <<- EOF
+  *********************************************************
+  Building deb package ${pkg} 
+  *********************************************************
+EOF
+  ./package.sh
+  _move_debs
+  echo "Done."
+}
+
 function _make_ros_deb() {
-  echo "Creating deb package ${1} ..."
+  local pkg=$1
+  cat <<- EOF
+  *********************************************************
+  Building ROS deb package ${pkg} 
+  *********************************************************
+EOF
   GIT_VER=0~dirty$(git log --date=format:%Y%m%d --pretty=~git%cd.%h -n 1)
-  build_dir=$(mktemp -d)
-  cp -r . ${build_dir}/package
-  pushd ${build_dir}/package > /dev/null
   _execute_build
   _move_debs
-  popd > /dev/null
-  rm -rf ${build_dir}
   echo "Done."
 }
 
+pushd ../tools/Fast-DDS-Gen
```

Comment:
This package is not needed in the release. Do we want to have it here?

Reply:
Fast-DDS-Gen is a build dependency to other fog-sw packages

Reply:
ok.